bin/Cargo.toml

```
@@ -20,9 +20,10 @@ serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0.66"
 tokio = { version = "1", features = ["full"] }
-zkevm = { path = "../zkevm" }
+zkevm = { path = "../zkevm", features = ["tachyon"] }
```

Why does it need `features = ["tachyon"]`?

```suggestion
zkevm = { path = "../zkevm" }
```